backend/src/auth/auth.module.ts

```diff
@@ -0,0 +1,19 @@
+import { Module } from '@nestjs/common';
+import { AuthService } from './auth.service';
+import { AuthController } from './auth.controller';
+import { JwtModule } from '@nestjs/jwt';
+import { LocalStrategy } from './strategy/local.strategy';
+import { JwtStrategy } from './strategy/jwt.strategy';
+import { PrismaService } from '../prisma/prisma.service';
+
+@Module({
+  imports: [
+    JwtModule.register({
+      secret: process.env.JWT_SECRET || 'yourDefaultSecretKey',
```

Comment:
you can use configService to get env
https://docs.nestjs.com/techniques/configuration